### ISSUE TITLE
Autopsy Starts and hangs while starting modules 

### ISSUE THREAD

**Issue by swalker6663** [sleuthkit/autopsy#6980]
So I have everything loaded and all the many steps met however when it starts I get the splash screen but nothing else happens. ive scoured the internet but I cannot find any similar results. PLEASE HELP.

OS- Parrot
Running on both a virtual box OS and the same results on a dedicated machine with a full Parrot OS

![image](https://user-images.githubusercontent.com/75349397/118735652-20838b00-b7fe-11eb-88fe-077be74d4f3b.png)
![hanging](https://user-images.githubusercontent.com/75349397/118735728-3d1fc300-b7fe-11eb-9e22-1bc8a88136de.png)

**Comment by jknyght9** [comment]
Try running it with the `--nosplash` argument. You can view the help by running `autopsy --help`. I got it to work temporarily.

**Comment by swalker6663** [comment]
So you’re says no run ./autopsy —nosplash

**Comment by jknyght9** [comment]
I dont know why it is hanging, but when I ran `autopsy --nosplash` the GUI came up and I was ready to work a case. Looks like there is a bug in either the splash screen or the module loader for 4.18 in Linux.

**Comment by swalker6663** [comment]
> I dont know why it is hanging, but when I ran `autopsy --nosplash` the GUI came up and I was ready to work a case. Looks like there is a bug in either the splash screen or the module loader for 4.18 in Linux.

It worked the issue is that there is a confirmation screen behind the splash when it starts that you cannot get too. Thank you for your help!! SOOOOOOOO many hours wasted for such a simple and undocumented problem.

**Comment by mh4ckt3mh4ckt1c4s** [comment]
Hello, any news for a fix ? I'm still getting this on the latest version, built from source. As I'm running it with `ant run` I can't / don't know how to use the `--nosplash` bypass. I'm on Archlinux with i3.

**Comment by swalker6663** [comment]
Yes there is a window hiding behind the splash picture once you click ok
everything worked.

**Comment by GregoriousTheBrave** [comment]
> Hello, any news for a fix ? I'm still getting this on the latest version, built from source. As I'm running it with `ant run` I can't / don't know how to use the `--nosplash` bypass. I'm on Archlinux with i3.

As a workaround, I managed to get in by using alt-tab to make the splash screen the active window, then pressing Alt-F4 to presumably close the hidden dialog box.

**Comment by mh4ckt3mh4ckt1c4s** [comment]
> As a workaround, I managed to get in by using alt-tab to make the splash screen the active window, then pressing Alt-F4 to presumably close the hidden dialog box.

Thanks for your answer. Thing is, using i3 doesn't allow me to do that. I'm stuck on the splash image no matter what, and it doesn't seem to be a dialog box or I can't access it with i3...

**Comment by dominikborkowski** [comment]
This issue continues to appear in 4.19.2. What exactly is that pop-up window setting? We'd like to be able to have a silent installer, but as it stands the first execution results in this problem.

In our case we use XFCE, and there is no indication there is another window.

**Comment by dominikborkowski** [comment]
Side note, here's the window in question, that's never seen due to the splash screen: 
<img width="442" alt="image" src="https://user-images.githubusercontent.com/3235494/190031015-da5159e2-35f4-4ae8-8a87-fb3bca0e6d4f.png">

**Comment by N3ar** [comment]
This spash screen issue is still relevant on Ubuntu 22.04 LTS as of Jan 2025.

**Comment by D3vil0p3r** [comment]
Any news on this? It is relevant to all Linux distributions (I am using NixOS).
